packages/core/content-releases/admin/src/index.ts

```diff
@@ -11,7 +11,10 @@ import type { Plugin } from '@strapi/types';
 const admin: Plugin.Config.AdminInput = {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   register(app: any) {
-    if (window.strapi.features.isEnabled('cms-content-releases')) {
+    if (
+      window.strapi.features.isEnabled('cms-content-releases') &&
```

Comment:
Any reason for the 2 keys to be different ? can't we make them consistent ?

Reply:
I think those keys are dictated by the cloud team because they manage them on the license registry cc @nathan-pichon?

we could ofc change the object to use that key – but it goes against how we normally write objects and possibly won't work with the ENV RFC?

Reply:
Well, I don't know what is `cms-content-releases` for. So I will need a bit of time to investigate what it is used for.